queries/dune_v2/period_slippage.sql

```diff
@@ -163,18 +164,18 @@ incoming_and_outgoing as (
         case
             when t.symbol = 'ETH' then 'WETH'
             when t.symbol is not null then t.symbol
-            else token
+            else cast(token as varchar)
```

Comment:
Nit: keeping table names prepended makes it easier to read from which table a specific column comes from.

```suggestion
            else cast(t.token as varchar)
```